dietpi/dietpi-software

```diff
@@ -1421,6 +1421,13 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DESC[$software_id]='validating, recursive, caching DNS resolver'
 		aSOFTWARE_CATX[$software_id]=13
 		aSOFTWARE_DOCS[$software_id]='https://dietpi.com/docs/software/dns_servers/#unbound'
+		
+		software_id=171
```

Comment:
ID conflict with [frp](https://github.com/MichaIng/DietPi/pull/4411/files) 😉.

Reply:
Which one you suggest?

Reply:
`126` became free due to LibSSL1.0.0 removal.

Reply:
done